src/components/ec-dropdown-search/ec-dropdown-search.spec.js

```diff
@@ -1,40 +1,256 @@
-import { mount } from '@vue/test-utils';
-import MutationObserver from 'mutation-observer';
-import EcDropdownSearch from '@/components/ec-dropdown-search/ec-dropdown-search.vue';
-
-global.MutationObserver = MutationObserver;
+import Vue from 'vue';
+import { mount, createLocalVue } from '@vue/test-utils';
+import EcDropdownSearch from './ec-dropdown-search.vue';
 
 describe('EcDropdownSearch', () => {
-  it('Should render as expected', () => {
-    const wrapper = mount(EcDropdownSearch, {
-      propsData: {
-        items: [
-          {
-            disabled: false,
-            active: true,
-            disabledReason:
-              'This account is not active at the moment. Please e-mail us at solutions@ebury.com or contact your relationship manager at +44 (0) 207 197 2421.',
-            id: 'EBPCLI00007',
-            text: 'Ebury Demo 2',
-          },
-          {
-            disabled: false,
-            active: false,
-            disabledReason:
-              'This account is not active at the moment. Please e-mail us at solutions@ebury.com or contact your relationship manager at +44 (0) 207 197 2421.',
-            id: 'EBPCLI00004',
-            text: 'Ebury Demo',
-          },
-        ],
-        user: {
-          clientName: 'Ebury Demo 2',
-          email: 'ebury.demo2@ebury.com',
-          gravatar:
-            'https://www.gravatar.com/avatar/e07fd6efc70ccc63bbc3a3e27b81b29e?d=mm&s=200',
-        },
+  const MockedEcTooltipDirective = {
+    bind(el, { value }) {
+      if (value.content) {
+        el.setAttribute('mocked-tooltip-content', value.content);
+        el.setAttribute('mocked-tooltip-placement', value.placement || '');
+      }
+    },
+  };
+
+  function mountDropdownSearch(props, mountOpts) {
```

Comment:
beautiful, I've followed this pattern before and definitely see it as the way to go

Reply:
I also like it, lots of repetition is removed and tests are more cleaner.